internal/service/vmservice/vm.go

```diff
@@ -191,16 +191,20 @@ func reconcileVirtualMachineConfig(ctx context.Context, machineScope *scope.Mach
 	if machineScope.ProxmoxMachine.Spec.Network != nil && shouldUpdateNetworkDevices(machineScope) {
```

Comment:
please also carefully change `extractNetworkModelAndBridge()` so it returns the right thing when MTU is not set.
maybe also it can return a struct instead.